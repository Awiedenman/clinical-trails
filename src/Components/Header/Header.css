* {
  margin: 0;
}

header {
  height: 162px;
  background-color: #61a6f0;
}

h1 {
  left: 20px;
  position: relative;
  top: 65px;
  color: #fff;
  font-size: 70px;
  text-align: left;
}

.navlink {
  position: absolute;
  right: 15px;
  top: 10px;
  color: #fff;
  text-decoration: none;
  
}

.navlink:hover {
  color: #ff5f58;
  
}