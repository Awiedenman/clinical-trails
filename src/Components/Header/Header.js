import React from 'react';
import { NavLink } from 'react-router-dom';
import './Header.css'

export const Header = () => {

  return(
    <header>
      <h1>Clinical Trials</h1>
      <NavLink className='navlink' to='signup/login'> SignUp/ LogIn</NavLink>
    </header>
  )
}