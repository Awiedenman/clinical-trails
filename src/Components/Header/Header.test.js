import { Header } from './Header';
import { shallow } from 'enzyme';
import React from 'react';

describe('Header', () =>{
  let wrapper;

  it('matches the snapshot', () => {
    wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  })
})
