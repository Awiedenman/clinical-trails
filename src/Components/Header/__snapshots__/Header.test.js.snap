// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Clinical Trials
</h1>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="navlink"
          to="signup/login"
>
           SignUp/ LogIn
</NavLink>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Clinical Trials",
        },
        "ref": null,
        "rendered": "Clinical Trials",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": " SignUp/ LogIn",
          "className": "navlink",
          "to": "signup/login",
        },
        "ref": null,
        "rendered": " SignUp/ LogIn",
        "type": [Function],
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Clinical Trials
</h1>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="navlink"
            to="signup/login"
>
             SignUp/ LogIn
</NavLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Clinical Trials",
          },
          "ref": null,
          "rendered": "Clinical Trials",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": " SignUp/ LogIn",
            "className": "navlink",
            "to": "signup/login",
          },
          "ref": null,
          "rendered": " SignUp/ LogIn",
          "type": [Function],
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
